.pre-commit-config.yaml

```diff
@@ -53,15 +53,12 @@ repos:
         files: ^pyproject\.toml$
     -   id: uv-sync
 
-# -   repo: https://github.com/pre-commit/mirrors-mypy
-#     rev: v1.14.0
-#     hooks:
-#     -   id: mypy
-#         additional_dependencies:
-#           - types-requests
-#           - types-setuptools
-#           - pydantic
-#         args: [--ignore-missing-imports]
+-   repo: local
+    hooks:
+    -   id: mypy
+        name: mypy
+        entry: bash -c 'uv sync --extra dev && uv run mypy'
```

Comment:
can this just be `uv run mypy`? see the pre-commit entry added by @bbrowning as an example.